src/server_manager/web_app/ui_components/outline-region-picker-step.ts

```diff
@@ -152,17 +153,15 @@ export class OutlineRegionPicker extends LitElement {
           return html`
           <input type="radio" id="card-${index}" name="city" value="${index}" ?disabled="${!option.available}" .checked="${this.selectedIndex === index}" @change="${this._locationSelected}">
           <label for="card-${index}" class="city-button">
-            <div class="card-header">
-              <div class="tag">
-                ${option.markedLowCost ? this.localize('region-lowest-cost') : ''}
-              </div>
-            </div>
             <div class="flag-overlay">
               <img class="flag" src="${this._flagImage(option)}">
             </div>
             <div class="geo-name">${getShortName(option.cloudLocation, this.localize)}</div>
             <div class="country-name">${option.cloudLocation.location?.countryIsRedundant() ? '' :
                 localizeCountry(option.cloudLocation.location, this.language)}</div>
+            ${option.markedLowCost ?
```

Comment:
Please rename the fields in code to reflect the new wording.

Reply:
Done